R/gset-gsea.r

```diff
@@ -84,7 +84,7 @@ gmt2mat <- function(gmt, max.genes = -1, ntop = -1, sparse = TRUE,
 #'
 mat2gmt <- function(mat) {
   idx <- Matrix::which(mat != 0, arr.ind = TRUE)
-  gmt <- tapply(rownames(idx), idx[, 2], list)
+  gmt <- tapply(as.character(rownames(idx)), idx[, 2], list)
```

Comment:
Any particular reason for this type cast? Just curious, I assume it solves some edge case.